packages/connect-web/src/impl/core-in-suite-desktop.ts

```diff
@@ -117,19 +115,14 @@ export class CoreInSuiteDesktop implements ConnectFactoryDependencies<ConnectSet
             );
 
             if (!response) {
-                return {
-                    success: false,
-                    payload: { error: 'No response', code: 'Desktop_ConnectionMissing' },
-                };
+                throw ERRORS.TypedError('Desktop_ConnectionMissing', 'No response');
             }
 
             return response;
         } catch (err) {
             return {
                 success: false,
-                payload: {
-                    error: err.message,
-                },
+                payload: ERRORS.serializeError(err),
```

Comment:
I got this 
<img width="264" alt="image" src="https://github.com/user-attachments/assets/90a0ab12-c159-4ebc-916e-c56a9e5d2aa6" />
 
<img width="554" alt="image" src="https://github.com/user-attachments/assets/27f1312d-72c8-4a27-8440-307b00dceb87" />

which does not look serialized 🤔